app/views/play/level/PlayLevelView.coffee

```diff
@@ -668,6 +675,18 @@ module.exports = class PlayLevelView extends RootView
       ModalClass = CourseVictoryModal
       options.courseInstanceID = utils.getQueryVariable('course-instance') or utils.getQueryVariable('league')
     ModalClass = PicoCTFVictoryModal if window.serverConfig.picoCTF
+
+    # Progress through the capstone stages
+    if additionalGoals
+      options.capstoneStage = @session.capstoneStage
+      options.remainingGoals = @goalManager.getRemainingGoals()
+      if options.remainingGoals.length > 0
+#        ModalClass = CapstoneProgressModal
+        ModalClass = GameDevVictoryModal
```

Comment:
Is this modal added just for testing?

Reply:
Yes - the UX/UI and progression is still unknown, so I just used a simple dialog for the time being. The two new dialogs may or may not be used depending on which direction UX/UI take.

Reply:
Got it, so are we planning to remove them before merging to master? We can add a comment or a console message as a placeholder until we know more (and maybe a wrap-up task for later when we know more)

Reply:
This has been updated with progression to a cinematic place holder page and back again to keep playing the level 👍